pybullet_tools/utils.py

```diff
@@ -345,7 +349,7 @@ def named_tuple(name, fields, defaults=None):
         NT.__new__.__defaults__ = defaults
     return NT
 
-class OrderedSet(collections.OrderedDict, collections.MutableSet):
+class OrderedSet(collections.OrderedDict, MutableSet):
```

Comment:
I also just imported `OrderedDict` in line 26 so it could just read:

```suggestion
class OrderedSet(OrderedDict, MutableSet):
```